src/Table/sgds-table.ts

```diff
@@ -24,6 +25,10 @@ export class SgdsTable extends SgdsElement {
 
   @state() sortColumn: number | null = null;
   @state() sortDirection: "asc" | "desc" = "asc";
```

Comment:
as there are only 2 ways to sort , it will be cleaner if use a boolean for `sortDirection` e.g. 
`  @state() sortAsc = true;`

then the code on line 36 dont need to evaluate so long  -->  `this.sortDirection = this.sortDirection === "asc" ? "desc" : "asc";`

it can be like this `this.sortAsc = !this.sortAsc` 
shorter and easier to read

then all the lines with ternary operator instead of `this.sortDirection === "asc" ? ...'`
 --> `this.sortAsc ?  ...`